### PR TITLE
Google analytics

### PR DESCRIPTION
**Pull request by exp-gle**
This should initialize google analytics, track page views, and has an example of an event tracker.

### CODE REVIEW

File: pages/utils/google-analytics/ga-track.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import ReactGA from 'react-ga';
+
+const isProd = true; //TODO - Track only production analytics otherwise dev and test would be tracked as well
```
Comment (exp-gle) [inline pages/utils/google-analytics/ga-track.js]: Need to figure out how to differentiate between different environments

Comment (JeffBeckstrom) [reply pages/utils/google-analytics/ga-track.js]: I should be able to add a filter on GA that will let me differentiate traffic from test.affordableseattle.org and affordableseattle.org.

Comment (exp-gle) [reply pages/utils/google-analytics/ga-track.js]: ok, i'll remove the that isProd for now and you can play around with the metrics.

**Review by JeffBeckstrom — COMMENTED** [review]
Looks good. Thanks for removing the isProd flag for now.

**Review by JeffBeckstrom — APPROVED** [review]
GA additions approved

**Pull request merged**